apps/addons/cron.py

```diff
@@ -453,9 +453,7 @@ def reindex_addons(index=None, addon_type=None):
     from . import tasks
     ids = (Addon.objects.values_list('id', flat=True)
```

Comment:
You're right, it does. With `objects`, it fixes the `disabled_by_user` cases but not the unlisted cases. Nice catch.

Don't think we need unfiltered, since the hook for deleted add-ons never stopped working.